src/inmanta/command.py

```diff
@@ -66,6 +52,7 @@ def add(
         parser_config: Optional[ParserConfigType],
         require_project: bool = False,
         aliases: List[str] = [],
+        verbose_opt_out: bool = False,
```

Comment:
I find this a rather strange name. Wouldn't something like `add_verbosity_flag: bool = True` be more to the point? Not a strong change request.